SimPEG/Examples/EM_FDEM_Mu_Inversion.py

```diff
@@ -0,0 +1,181 @@
+from SimPEG import (
+    np, Mesh, Maps, Utils, DataMisfit, Regularization,
+    Optimization, Inversion, InvProblem, Directives
+)
+from SimPEG.EM import FDEM, TDEM, mu_0
+import matplotlib.pyplot as plt
+import matplotlib
+try:
+    from pymatsolver import PardisoSolver as Solver
+except ImportError:
+    from SimPEG import SolverLU as Solver
+
+
+def run(plotIt=True):
+    """
+    1D FDEM Mu Inversion
+    ====================
+
+    1D inversion of Magnetic Susceptibility from FDEM data assuming a fixed
+    electrical conductivity
+
+    """
+
+    # Set up cylindrically symmeric mesh
+    cs, ncx, ncz, npad = 10., 15, 25, 13  # padded cyl mesh
+    hx = [(cs, ncx), (cs, npad, 1.3)]
+    hz = [(cs, npad, -1.3), (cs, ncz), (cs, npad, 1.3)]
+    mesh = Mesh.CylMesh([hx, 1, hz], '00C')
+
+    # Geologic Parameters model
+    layerz = np.r_[-100., -50.]
+    layer = (mesh.vectorCCz >= layerz[0]) & (mesh.vectorCCz <= layerz[1])
+    active = mesh.vectorCCz < 0.
+
+    # Electrical Conductivity
+    sig_half = 1e-2  # Half-space conductivity
+    sig_air = 1e-8  # Air conductivity
+    sig_layer = 1e-2  # Layer conductivity
+    sigma = np.ones(mesh.nCz)*sig_air
+    sigma[active] = sig_half
+    sigma[layer] = sig_layer
+
+    # mur - relative magnetic permeability
+    mur_half = 1.
+    mur_air = 1.
+    mur_layer = 2.
+    mur = np.ones(mesh.nCz)*mur_air
+    mur[active] = mur_half
+    mur[layer] = mur_layer
+
+    mtrue = mur[active]
+
+    # Maps
+    actMap = Maps.InjectActiveCells(mesh, active, mur_air, nC=mesh.nCz)
+    surj1Dmap = Maps.SurjectVertical1D(mesh)
+    murMap = Maps.Mu_relative(mesh)
+
+    # Mapping
+    muMap =  murMap * surj1Dmap * actMap
```

Comment:
the tests are testing all formulations, so we will hit examples where the mui map is used. The tests could be expanded to include setting both mu and mui maps. I think generally, one would invert for 'mu', thinking in terms of mu-inverse is pretty unnatural.